multi-node/aws/pkg/config/config_test.go

```diff
@@ -253,3 +253,59 @@ func TestIsSubdomain(t *testing.T) {
 	}
 
 }
+
+func TestReleaseChannel(t *testing.T) {
+
+	validConfigs := []struct {
+		conf    string
+		channel string
+	}{
+		{
+			conf: `
+releaseChannel: alpha
+`,
+			channel: "alpha",
+		},
+		{
+			conf: `
+releaseChannel: beta
+`,
+			channel: "beta",
+		},
+	}
+
+	invalidConfigs := []string{
+		`
+#TODO(chom): move this to validConfigs when stable is supported
```

Comment:
We need the newlines in the beginning and end, so that's why I have the multi-lines definitions.

I have the TODO in the test case so, if the test ever fails, we'll have a reminder there when it prints out the config block as part of error message.